hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -72,6 +72,7 @@ const (
 
 	nodePoolAnnotationPlatformMachineTemplate = "hypershift.openshift.io/nodePoolPlatformMachineTemplate"
 	nodePoolCoreIgnitionConfigLabel           = "hypershift.openshift.io/core-ignition-config"
+	nodePoolOperatorCreatedConfigLabel        = "hypershift.openshift.io/operator-generated-config"
```

Comment:
the name of the variable refers to "created" and the value to "generated", can we use a single term?
Also can we please refer to what generates this? e.g `hypershift.openshift.io/nto-config`

Reply:
also can you be consistent for variable naming and value with 
tunedConfigKey      = "tuned"
tunedConfigMapLabel = "hypershift.openshift.io/tuned-config"

and group all the variable declarations together?

Reply:
can you also add a comment to reconcileTunedConfigMap elaborating that it is just mirroring the config into the control plane namespace for the nto operator consumption (which then mirror them into the guest cluster for the daemon consumption?).

Reply:
Thanks, I think I addressed all of these comments. I made the label NTO-specific, and grouped it with the other tuned* labels, and added the comment you requested.